### PR TITLE
lens: init at 3.6.7

### PR DESCRIPTION
**Pull request by dbirks**
###### Motivation for this change

Wanted to contribute a package I use several times a week.

And this should close issue #88168.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/cluster/lens/default.nix
Comment (SuperSandro2000) [inline pkgs/applications/networking/cluster/lens/default.nix:4]:
```suggestion
{ lib, fetchurl, appimageTools }:
```

File: pkgs/applications/networking/cluster/lens/default.nix
Comment (SuperSandro2000) [inline pkgs/applications/networking/cluster/lens/default.nix:14]:
```suggestion
    name="${name}.AppImage";
```

File: pkgs/applications/networking/cluster/lens/default.nix
Comment (dbirks) [reply pkgs/applications/networking/cluster/lens/default.nix]: Thank you, fixed.

File: pkgs/applications/networking/cluster/lens/default.nix
Comment (dbirks) [reply pkgs/applications/networking/cluster/lens/default.nix]: Fixed this too :+1:

**Comment by dbirks** [comment]
Also removed one whitespace character to hopefully fix the `Checking EditorConfig` error:
```
pkgs/applications/networking/cluster/lens/default.nix:
	24: Trailing whitespace
```

**Comment by roberth** [comment]
Nothing happens when I click any of the documentation links on the right pane when adding a cluster, or when I click the items in the help menu (except What's new).
Stderr doesn't show messages about this, but it seems like it can't browser windows (unlike `xdg-open https://nixos.org` which does work in my environment)

**Comment by dbirks** [comment]
Thanks for building and trying it out!
I did a fresh rebuild, and removed my `~/.config/Lens` directory to make sure I had a clean environment, but unfortunately I can't reproduce those problems with the links not opening. I'm on GNOME 3 for background.

**Comment by roberth** [comment]
I'm also on GNOME 3. My normal user doesn't get any logs but I've tried it on a clean user with firefox in its profile (`nix-env -i`) and got this when clicking links:

<details><summary>stderr</summary>

```
/nix/store/2jysm3dfsgby5sw5jgj43qjrb5v79ms9-bash-4.4-p23/bin/bash: symbol lookup error: /usr/lib/libc.so.6: undefined symbol: _dl_fatal_printf, version GLIBC_PRIVATE
/usr/bin/xdg-open: line 883: x-www-browser: command not found
/nix/store/2jysm3dfsgby5sw5jgj43qjrb5v79ms9-bash-4.4-p23/bin/bash: symbol lookup error: /usr/lib/libc.so.6: undefined symbol: _dl_fatal_printf, version GLIBC_PRIVATE
/usr/bin/xdg-open: line 883: iceweasel: command not found
/usr/bin/xdg-open: line 883: seamonkey: command not found
/usr/bin/xdg-open: line 883: mozilla: command not found
/usr/bin/xdg-open: line 883: epiphany: command not found
/usr/bin/xdg-open: line 883: konqueror: command not found
/usr/bin/xdg-open: line 883: chromium: command not found
/usr/bin/xdg-open: line 883: chromium-browser: command not found
/usr/bin/xdg-open: line 883: google-chrome: command not found
/usr/bin/xdg-open: line 883: www-browser: command not found
/usr/bin/xdg-open: line 883: links2: command not found
/usr/bin/xdg-open: line 883: elinks: command not found
/usr/bin/xdg-open: line 883: links: command not found
/usr/bin/xdg-open: line 883: lynx: command not found
/usr/bin/xdg-open: line 883: w3m: command not found
xdg-open: no method available for opening 'https://k8slens.dev/licenses/eula.md'
```

</details>

I'm running NixOS 20.09.

**Comment by dbirks** [comment]
I'm on different system now, but I haven't been able to get the same errors as you yet. :slightly_frowning_face: 

I do have my configuration.nix in git now if you want to compare.:
https://github.com/dbirks/dotfiles/blob/master/nixos/etc/nixos/configuration.nix

I wonder if something in this section in `services` is what's making the difference:

```
    gnome3 = {
      core-os-services.enable = true;
      core-shell.enable = true;
      core-utilities.enable = true;
    };
```

**Review by roberth — APPROVED** [review]
Links are also unreliable in vscode for me, so it's unlikely to be a lens issue and certainly not a reason to block this pr.
LGTM!

**Comment by dbirks** [comment]
Thank you! :surfer:

**Comment by MaryamTavakkoli** [comment]
Does this work only with Gnome? (I have Plasma and couldn't install lens)

**Comment by roberth** [comment]
We've tested with gnome, but it shouldn't be gnome-specific. Could you provide the error log?

**Comment by dbirks** [comment]
I hit an error that might be the same as yours:
```
error: undefined variable 'lens' at /home/david/dotfiles/nixos/etc/nixos/configuration.nix:65:5
```
I think that's because the `nixos-unstable` channel has been backed up with a build problem since 9 days ago, before this PR was merged: https://status.nixos.org/

Looks like a fix is on its way though.

**Comment by yurrriq** [comment]
I'm not too familiar with the AppImage wrapper. How would I, for example, get `kubelogin` (another Nix package) on the `$PATH`?

**Pull request merged**
